ui/selectors/transactions.js

```diff
@@ -643,8 +627,12 @@ const TRANSACTION_APPROVAL_TYPES = [
 ];
 
 export function hasTransactionPendingApprovals(state) {
+  const unapprovedTxRequests = getApprovalRequestsByType(
+    state,
+    ApprovalType.Transaction,
+  );
   return (
-    hasUnapprovedTransactionsInCurrentNetwork(state) ||
+    unapprovedTxRequests.length > 0 ||
```

Comment:
since we queue requests on different dapps which can be on different networks now we don't want to restrict this boolean check to only transactions that match the currently selected network. I'm not actually sure how there could have been pending transactions on different networks previously that necessitated this check 🤔... but I might be forgetting something